cowprotocol/explorer
# [Tokens Table] Sort by 24h volume

# Summary

Closes #210 

- Refactor code to get tokens table info in just one query to the subgraph.
- Sort the tokens table by 24h Volume instead of Total Volume.

<img width="1251" alt="Screen Shot 2022-09-14 at 00 38 18" src="https://user-images.githubusercontent.com/11525018/190054577-4d8917c6-5c7c-4a07-bcee-562adf627cfb.png">


# To Test

1.  Open the `home` page and wait for the tokens table to load.
*  You'll see the tokens table is now sorted by 24h volume.
